### ISSUE TITLE
Latest NAACCR vocabularies

### ISSUE THREAD

**Issue by MelaniePhilofsky** [OHDSI/OncologyWG#303]
Hello all,

I am working with a cancer registry dataset based on this data dictionary: http://datadictionary.naaccr.org/default.aspx?c=10&Version=21#1390

Not all of the source codes are mapped to concept_ids.

**Comment by dimshitc** [comment]
Yeah, they already have version 21, while we ingested NAACCR version 18. It will be put in a vocabulary team's backlog.
Thanks for reporting!
